File: MyLanguageDB/src/main/java/ru/herooo/mylanguagedb/repositories/workouttype/WorkoutTypes.java
package ru.herooo.mylanguagedb.repositories.workouttype;

public enum WorkoutTypes {
    RANDOM_WORDS(1, "random_words"),
    COLLECTION_WORKOUT(2, "collection_workout")

    ;

    public final long ID;
    public final String CODE;

    WorkoutTypes(long id, String code) {
        this.ID = id;
        this.CODE = code;
    }
}

File: MyLanguageWeb/src/main/resources/static/js/classes/api/workout_types/workout_types.js
class WorkoutType {
    ID;
    CODE;

    constructor(id, code) {
        this.ID = id;
        this.CODE = code;
    }
}

export class WorkoutTypes {
    RANDOM_WORDS = new WorkoutType(1, "random_words");
    COLLECTION_WORKOUT = new WorkoutType(2, "collection_workout");
}